packages/vue/src/stories/releases/v0.10.0/migration.stories.mdx

```diff
@@ -394,4 +389,33 @@ It was replaced by grid styling set directly in the inline CSS like this:
       <slot />
     </div>
   </footer>
-```
\ No newline at end of file
+```
+
+## Atoms refactored
+
+As the better web performance is one of our main goals, we refactored components without additional logic into functional components.
```

Comment:
"As the better web performance is one of our main goals, the following components have been changed into functional components."

Remove "This is the list of changed components:" :)

Reply:
Ok fixed :)